products/sle/main.pm

```diff
@@ -658,6 +658,7 @@ sub load_consoletests() {
             }
             elsif (!get_var("ZDUP") and !check_var('VERSION', '12')) {    # zypper and sle12 doesn't do upgrade or installation snapshots
                 loadtest "console/installation_snapshots.pm";
+                loadtest "console/snapper_cleanup.pm";
```

Comment:
what about a corresponding TW test? Please check if it works with a current TW snapshot and then add to the main.pm there